BranchMonsterFactory/ViewControllers/MonsterViewerViewController.m

```diff
@@ -124,12 +124,13 @@ -(IBAction)shareSheet:(id)sender {
     commerceEvent.currency = @"USD";
     BNCProduct* branchester = [BNCProduct new];
     branchester.sku = [self.viewingMonster getMonsterName];
-    //branchester.name = [self.viewingMonster getMonsterName];
     commerceEvent.products = [NSArray arrayWithObject:branchester];
     
+    //[[Branch getInstance] userCompletedAction:BNCAddToCartEvent];
     [self.viewingMonster
      showShareSheetWithShareText:@"Share Your Monster!" completion:^(NSString * _Nullable activityType, BOOL completed) {
             if (completed) {
+                [[Branch getInstance] userCompletedAction:BNCAddToCartEvent];
```

Comment:
Can it look good now. Add to cart would be triggered when someone taps on the share button.